pkgs/development/libraries/libgpiod/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchgit, autoconf, autoconf-archive, automake, libtool, pkgconfig, kmod, enable-tools ? true }:
+
+stdenv.mkDerivation rec {
+  name = "libgpiod-${version}";
+  version = "2018-10-07";
+
+  src = fetchgit {
+    url = "git://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git";
+    rev = "4bf402d3a49336eacd33654441d575bd267780b8";
+    sha256 = "01f3jzb133z189sxdiz9qiy65p0bjqhynfllidbpxdr0cxkyyc1d";
+  };
+
+  buildInputs = [ kmod ];
+  nativeBuildInputs = [
+    autoconf
+    autoconf-archive
+    automake
+    libtool
+    pkgconfig
+  ];
+
+  configurePhase = ''
+    ./autogen.sh \
```

Comment:
Since `autogen.sh` does not anything special here,
I would prefer to not override `configurePhase` and just put
`autoReconfhook` into `nativeBuildInputs`. This has the advantage
that our configurePhase can add some fixes if necessary.
Additionally you can set `configureFlags` to pass `--enable-tools` etc.

https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/tree/autogen.sh